src/utils/api.js

```diff
@@ -15,6 +15,19 @@ export const getTopics = () => {
     })
 }
 
+export const getArticlesByTopic = (topic) => {
+    return nc_news.get(`/articles?topic=${topic}`).then(({ data }) => {
+
+        if(topic === 'cooking'){
+            return [data.articles[7], data.articles[17], data.articles[18]]
+        } else if (topic === 'football'){
+            return [data.articles[8], data.articles[11], data.articles[16]]
+        } else if (topic === 'coding'){
+            return [data.articles[19], data.articles[21], data.articles[31]]
+        }
+    })
+}
+
```

Comment:
I appreciate you didn't build the query into your BE, however this approach is very brittle.

Try fetching all articles and using a filter to find the articles with the topic you are looking for.

Reply:
Okay perfect. I gave that a try yesterday but wasn't sure how to filter each article like using array methods. Should I try that before the map?

Reply:
Ahh I just managed it, thanks!